docs/source/tutorial/set_up_experiment_tracking.md

```diff
@@ -174,6 +174,23 @@ You can now access, compare and pin your runs by toggling the `Compare runs` but
 
 ![](../meta/images/experiment-tracking_demo.gif)
 
+## View and compare plot data
+
+Experiment tracking in Kedro-Viz also supports the display and comparison of plots, such as Plotly and Matplotlib. Once you have [created your plots](../tutorial/visualise_pipeline.md#visualise-plotly-charts-in-kedro-viz) in your kedro-project, you must set the versioned flag to `true` within the project catalog to include them in experiment tracking.
+
+```
+# conf/base/catalog.yml
+
+reporting.confusion_matrix:
+  type: matplotlib.MatplotlibWriter
+  filepath: ${base_location}/08_reporting/confusion_matrix.png
+  versioned: true
+```
+
+Clicking on a plot will expand it. If in comparison view, expanding a plot will show all the plots in that view for them to be compared side-by-side.
+
+![](../meta/images/expand-plot-comparison-view.gif)
+
 Additionally, you can monitor the changes to metrics over time from the pipeline visualisation tab ![](../meta/images/pipeline_visualisation_icon.png). Clicking on any [MetricsDataset](https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.tracking.MetricsDataSet.html) node will open a side panel displaying how the metric value has changed over time.
```

Comment:
I think this should be moved to some sections above and not in the end under View and Compare Plots or maybe have it's own subtitle